modules/fedimint-wallet-client/src/lib.rs

```diff
@@ -215,6 +223,132 @@ impl ClientModuleInit for WalletClientInit {
             task_group: args.task_group().clone(),
         })
     }
+
+    /// Wallet recovery
+    ///
+    /// Query bitcoin rpc for history of addresses from last known used
+    /// addresses (or index 0 one) until MAX_GAP unused ones.
+    ///
+    /// Notably does not persist the progress of addresses being querried,
+    /// because it is not expected that it would take long enough to bother.
+    async fn recover(
```

Comment:
[nit] We can update the fedimint cli docs for restore to also include the wallet module

https://github.com/fedimint/fedimint/blob/9c9b1bda637222993bdbe5865e592cd6a37e26f2/fedimint-cli/src/client.rs#L172-L173